com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -12,282 +14,180 @@ namespace MLAPI.Prototyping
-        public Authority TransformAuthority = Authority.Server; // todo Luke mentioned an incoming system to manage this at the NetworkBehaviour level, lets sync on this
-
-        /// <summary>
-        /// The base amount of sends per seconds to use when range is disabled
-        /// </summary>
-        [SerializeField, Range(0, 120), Tooltip("The base amount of sends per seconds to use when range is disabled")]
-        public float FixedSendsPerSecond = 30f;
-
-        /// <summary>
-        /// TODO MTT-766 once we have per var interpolation
-        /// Enable interpolation
-        /// </summary>
-        [SerializeField, Tooltip("This requires AssumeSyncedSends to be true")]
-        public bool InterpolatePosition = true;
-
-        /// <summary>
-        /// TODO MTT-766 once we have per var interpolation
-        /// The distance before snaping to the position
-        /// </summary>
-        [SerializeField, Tooltip("The transform will snap if the distance is greater than this distance")]
-        public float SnapDistance = 10f;
-
-        /// <summary>
-        /// TODO MTT-766 once we have per var interpolation
-        /// Should the server interpolate
-        /// </summary>
-        [SerializeField]
-        public bool InterpolateServer = true;
+        private class NetworkState : INetworkSerializable
+        {
+            public bool InLocalSpace;
+            public Vector3 Position;
+            public Quaternion Rotation;
+            public Vector3 Scale;
 
-        /// <summary>
-        /// TODO MTT-767 once we have this per var setting. The value check could be more on the network variable itself. If a server increases
-        ///      a Netvar int by +0.05, the netvar would actually not transmit that info and would wait for the value to be even more different.
-        ///      The setting in the NetworkTransform would be to just apply it to our netvars when available
-        /// The min meters to move before a send is sent
-        /// </summary>
-        [SerializeField, Tooltip("The min meters to move before a send is sent")]
-        public float MinMeters = 0.15f;
+            public void NetworkSerialize(NetworkSerializer serializer)
+            {
+                serializer.Serialize(ref InLocalSpace);
+                serializer.Serialize(ref Position);
+                serializer.Serialize(ref Rotation);
+                serializer.Serialize(ref Scale);
+            }
+        }
 
         /// <summary>
-        /// TODO MTT-767 once we have this per var setting
-        /// The min degrees to rotate before a send is sent
+        /// TODO this will need refactoring
+        /// Specifies who can update this transform
         /// </summary>
-        [SerializeField, Tooltip("The min degrees to rotate before a send is sent")]
-        public float MinDegrees = 1.5f;
+        [FormerlySerializedAs("TransformAuthority"), Tooltip("Defines who can update this transform")]
```

Comment:
I don't think we need `FormerlySerializedAs` here as this was just introduced and never in a published version?